lib/src/notification/notification.component.scss

```diff
@@ -0,0 +1,288 @@
+@import "~@sebgroup/bootstrap/scss/variables";
+
+$small: 12px;
```

Comment:
the smaller font-size that SEB uses is 0.875rem = 14px, i dont think we should come up with our own font size